src/pages/guides/quickstart/index.md

```diff
@@ -15,46 +15,99 @@ keywords:
   - Getting started
   - Browser support
 title: Quickstart Guide
-description: This is the quickstart guide for Adobe Express Embed SDK
+description: This is the Quickstart Guide for Adobe Express Embed SDK
 contributors:
   - https://github.com/amandahuarng
   - https://github.com/nimithajalal
+  - https://github.com/undavide
 ---
+import "./styles.css";
+
 # Quickstart Guide
 
-This guide explains how to start using the Adobe Express Embed SDK in your own application.
+In this guide, you will learn how to get started with the Adobe Express Embed SDK.
 
 ## Step 1: Get an API Key
```

Comment:
Let us section this out as two ways to get an API Key.

1. **Using the Get Credentials feature**- In this, we can add the entire content of ## Step 1: Get an API Key

2. **Using the Dev Console**. - In this, we can add the content from the info block of **Creating a Project directly from the Adobe Developer Console**